.gitignore

```diff
@@ -9,3 +9,9 @@
 /captures
 .externalNativeBuild
 /.idea/caches/
+fastlane/key.json
+fastlane/report.xml
+fastlane/screenshots/*/*.png
+fastlane/screenshots/screenshots.html
+fastlane/screenshots/report.xml
+fastlane/metadata/
```

Comment:
Do you want to only filter top level or any sub-folder named fastlane with such content?

Reply:
I want to filter this specific content from the fastlane subfolder